File: src/main/java/com/beemer/fromis9server/album/dto/AlbumListDTO.java
package com.beemer.fromis9server.album.dto;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.Getter;
import lombok.Setter;

import java.time.LocalDate;

@Getter
@Setter
@JsonInclude(JsonInclude.Include.NON_NULL)
public class AlbumListDTO {
    private String albumName;
    private String albumType;
    private LocalDate releaseDate;
    private String colorPrimary;
    private String colorSecondary;
    private AlbumArtDTO albumArt;
    private AlbumDescriptionDTO albumDescription;
}

File: src/main/kotlin/com/beemer/unofficial/fromis9/album/dto/AlbumListDto.kt
package com.beemer.unofficial.fromis9.album.dto

data class AlbumListDto(
    val albumName: String,
    val type: String,
    val cover: String,
    val release: String,
    val colorMain: String,
    val colorPrimary: String,
    val colorSecondary: String
)